templates/index/dashboard.html.twig

```diff
@@ -2,56 +2,42 @@
 
 {% block content %}
 
-{% include 'index/dashboard.create.html.twig' with {form: form} %}
-
-<div class="row">
-
-    <div class="large-4 small-12 columns">
-
-        <h3><i class="fa fa-calendar"></i> Upcoming Talks</h3><hr>
-
-        {% for topic in next_topics %}
-        <div class="quiet one-line">
-            <i class="fa fa-clock-o"></i> {{ topic.lectureDate|date("Y-m-d") }}
-            <i class="fa fa-microphone"></i> {{ topic.speakers|join(", ") }}
-        </div>
-        <h5 class="one-line">
-            <a href="{{ path('topic-show', {id: topic.id}) }}">{{ topic.title }}</a>
-        </h5><br />
-        {% else %}
-            <div class="alert-box secondary">
-            no topics
-            </div>
-        {% endfor %}
-
-        <h3><i class="fa fa-clock-o"></i> Last Submissions</h3><hr>
-
-        {% for topic in last_submissions %}
-        <div class="quiet"><span title="{{ topic.createdAt|date }}" data-tooltip>{{ topic.createdAt|time_ago_in_words }}</span></div>
-        <h5 class="one-line">
-            <a href="{{ path('topic-show', {id: topic.id}) }}">{{ topic.title }}</a>
-        </h5><br />
-        {% else %}
-            <div class="alert-box secondary">
-            no topics
+    {% include 'index/dashboard.create.html.twig' with {form: form} %}
+
+    <div class="row" ng-controller="IndexController">
+
+        <div class="column">
+            <h3>Open Topics</h3>
+
+            <div class="row collapse">
+                <div class="large-4 columns">
+                    <select ng-model="order" ng-options="opt as opt.label for opt in sorting"></select>
```

Comment:
Yes, but you can switch to descending ordering, which would mean "lowest ranking", so at this point the option name is wrong. Therefore my suggestions above are neutral in regard to the sort order.